backend/redis/sender.c

```diff
@@ -316,4 +314,7 @@ void dbBE_Redis_sender_trigger( dbBE_Redis_context_t *backend )
   args->_backend = backend;
   args->_looping = 1;
   dbBE_Redis_sender( (void*) args );
+  dbBE_Redis_receiver( (void*) args );
+  dbBE_Redis_sender( (void*) args );
```

Comment:
why twice?

Reply:
Initial thought was: for multi-stage requests, the receiver creates new requests to send. Processing those right away could be beneficial. However, I abandoned this approach later (as you noticed) because a) it's detrimental to coalescing and b) the multi-stage requests are not considered critical path commands (think: nscreate/delete, directory & co)